app/uk/gov/hmrc/tariffclassificationfrontend/views/case_details.scala.html

```diff
@@ -30,7 +30,7 @@
             @case_nav(page, c)
 
             <div class="tabs__panel">
-                <div class="tabs__panel__content">
+                <div id="tab-content" class="tabs__panel__content">
```

Comment:
Our id naming strategy is usually `module_name-element`
So following the same pattern this would be `case_details-tab_content`